Source/Core/VideoCommon/RenderBase.h

```diff
@@ -133,6 +134,8 @@ class Renderer
 	// Max height/width
 	virtual int GetMaxTextureSize() = 0;
 
+	static Common::Event s_screenshotCompleted;
```

Comment:
Ok, good - that was my understanding of how it would be done using a `bool`, and I also don't want to do that. I guess we could get rid of the `bool`, but I dunno if that will cause problems elsewhere.